### PR TITLE
Update to contributing-organizations.md

### PR DESCRIPTION
**Pull request by jeremiah**
I added an example which the CHAOSS team may want to consider.

### CODE REVIEW

**Comment by germonprez** [comment]
Thanks @jeremiah -- will take a look!

**Comment by germonprez** [comment]
Cool. Is there source for this?

**Comment by sgoggins** [comment]
Lets try to review this pull request prior to the next WG-GMD meeting. It seems to have gotten lost in the shuffle. With apologies to @jeremiah !!
--sean

File: activity-metrics/contributing-organizations.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ An example implementation, for example a SQL or Elasticsearch query.
 
 ## 6. Known Implementations
 Examples of where and how metric is used. (include links to dashboard or location where metric is visible or is talked about having been used).
+Example: GENIVI contributors: https://docs.projects.genivi.org/kpi/2018_committers.html
```
[the remark this answers is not in the document]
Comment (sgoggins) [reply activity-metrics/contributing-organizations.md]: There's a lot of regular expression checking as well. A bit of commenting around the code example would be appreciated, but not doing that does not present an objection.

[the remark this answers is not in the document]
Comment (jeremiah) [reply activity-metrics/contributing-organizations.md]: Yes, I'll follow the formatting suggestion and comment on the regex -- thanks for the feedback!

[the remark this answers is not in the document]
Comment (jeremiah) [reply activity-metrics/contributing-organizations.md]: Were you thinking something like this @sgoggins? --> https://github.com/GENIVI/admin_and_reporting/blob/master/get_all_genivi_repos.sh

**Comment by BenLloydPearson** [comment]
Perhaps I'm missing something, but I'm not sure this example actually answers the question: What is the number of contributing organizations? This would be a single integer, not a list of companies ranked by contributions (this should be a separate entry in activity-metrics). I suppose you could count the number of organizations that are listed, but it's not clear if that list is a comprehensive list of all organizations that contribute.

**Comment by sgoggins** [comment]
> Perhaps I'm missing something, but I'm not sure this example actually answers the question: What is the number of contributing organizations? This would be a single integer, not a list of companies ranked by contributions (this should be a separate entry in activity-metrics). I suppose you could count the number of organizations that are listed, but it's not clear if that list is a comprehensive list of all organizations that contribute.

I think this is a great question!

**Comment by GeorgLink** [comment]
@jgbarah I believe your requested changes have been addressed?

**Comment by jgbarah** [comment]
> @jgbarah I believe your requested changes have been addressed?

Yes, you're right. I changed my review to approved.

**Comment by GeorgLink** [comment]
@jeremiah
Thank you for this contribution.

**Pull request merged**